GoudeauRampUp/Assets/Scripts/MoveP2.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoveP2 : MonoBehaviour
```

Comment:
Can you make the Move script one base script to prevent code duplication? Try making the base class with an update using something like this:

`protected string horizontalAxis = "Horizontal";`
`protected string verticalAxis = "Vertical";`
`Vector3 movement = new Vector3(Input.GetAxis(horizontalAxis), 0.0f, Input.GetAxis(verticalAxis));`

That way you can inherit from Move with MoveP2 and change the variables to "Horizontal2" and "Vertical2" and have no code duplication

Reply:
I fixed the other things you mentioned but I'm a little confused on how I should be doing this one. You want there to only be one move script?

Reply:
There should actually be 3 move scripts but with far less duplicated code. There should be a Move script that is basically one of the two existing ones, then a MoveP1 and MoveP2 script that inherits from this base one. In each inherited one the values for `horizontalAxis` and `verticalAxis` should be overridden to correspond to the player. 

I was thinking about this today, and an even better option would be to have only one move script with the axis names as private variables that can be set in the editor. This has the customizing power and cut down in code duplication of the other solution with the added bonus of there being only one class to keep track of.